src/hostname/hostnamed.c

```diff
@@ -1320,7 +1383,11 @@ static const sd_bus_vtable hostname_vtable[] = {
                                 SD_BUS_RESULT("s", json),
                                 method_describe,
                                 SD_BUS_VTABLE_UNPRIVILEGED),
-
+        SD_BUS_METHOD_WITH_ARGS("GetBiosVersion",
```

Comment:
please replace the string "bios" with "firmware" everywhere please.

BIOS is the PC firmware from the 1990s. Today we use UEFI instead of BIOS. Many people use BIOS and firmware interchangable, but it is a bit incorrect to call a modern UEFI fimware "bios", hence let's avoid that here.

Reply:
this also returns the version of old-style bios, not just uefi, so uefi_firmware wouldn't be accurate